Modules/Core/Common/include/itkHexahedronCell.hxx

```diff
@@ -581,9 +581,9 @@ HexahedronCell<TCellInterface>::EvaluateLocation(int &                     itkNo
   std::fill_n(x, Self::CellDimension, 0.0);
   for (unsigned int i = 0; i < Self::NumberOfPoints; i++)
   {
-    PointType pt = points->GetElement(m_PointIds[i]);
+    const PointType pt{ points->GetElement(m_PointIds[i]) };
 
-    for (unsigned int j = 0; j < Self::CellDimension; j++)
+    for (unsigned int j = 0; j < PointType::Dimension; j++)
```

Comment:
Good find -- certain compiler combinations were flagging this with a warning, if I recall correctly.